src/components/course/data/service.ts

```diff
@@ -59,39 +114,57 @@ export default class CourseService {
     };
   }
 
-  async fetchAllCourseRecommendations(activeCatalogs = []) {
-    const resp = await this.fetchCourseRecommendations()
-      .then(async (response) => {
+  async fetchAllCourseRecommendations(
+    activeCatalogs = []
+  ): Promise<CourseRecommendation[]> {
+    const resp = await this.fetchCourseRecommendations().then(
+      async (response) => {
         const {
           all_recommendations: allRecommendations,
           same_partner_recommendations: samePartnerRecommendations,
         } = response.data;
 
         // handle no recommendations case
-        if (allRecommendations.length < 1 && samePartnerRecommendations.length < 1) {
+        if (
+          allRecommendations.length < 1 &&
+          samePartnerRecommendations.length < 1
```

Comment:
[curious] What is causing all these unnecessary splitting of lines into multiple lines throughout this file and PR? In many instances, the original lines are well under the max line length of 120 per the ESLint config specified in `@edx/frontend-build`, but they are getting split into multiple lines? Not trying to be nitpicky on formatting, but these formatting changes feel a bit erroneous.